public/claimlist.php

```diff
@@ -0,0 +1,313 @@
+<?php
+
+use RA\ClaimFilters;
+use RA\ClaimSetType;
+use RA\ClaimSorting;
+use RA\ClaimSpecial;
+use RA\ClaimStatus;
+use RA\ClaimType;
+use RA\Permissions;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../lib/bootstrap.php';
+
+authenticateFromCookie($user, $permissions, $userDetails);
+
+$defaultFilter = ClaimFilters::Default; // Show all active claims
+$defaultSorting = ClaimSorting::ClaimDateDescending; // Sort by claim date
+$maxCount = 50;
+$offset = 0;
+$totalClaims = 0;
+
+$errorCode = requestInputSanitized('e');
+$type = requestInputSanitized('t', 0, 'integer');
+$username = requestInputSanitized('u', null);
+$claimFilters = requestInputSanitized('f', $defaultFilter, 'integer');
+$sortType = requestInputSanitized('s', $defaultSorting, 'integer');
+$gameID = requestInputSanitized('g', null, 'integer');
+$limit = requestInputSanitized('c', $maxCount, 'integer');
+$offset = requestInputSanitized('o', $offset, 'integer');
+
+if ($type == 0) { // Get general data
+    $claimData = getFilteredClaimData($gameID, $claimFilters, $sortType, false, $username, false, $offset, $limit);
+    $totalClaims = getFilteredClaimData($gameID, $claimFilters, $sortType, false, $username, true);
+} else { // Get expiring data
+    $claimData = getFilteredClaimData(0, $defaultFilter, ClaimSorting::FinishedDateAscending, true, $username, false); // Active sorted by expiring
+}
+$activeClaimCount = getActiveClaimCount();
+
+if (!empty($gameID)) {
+    getGameTitleFromID($gameID, $gameTitle, $consoleID, $consoleName, $forumTopic, $gameData);
+}
+
+sanitize_outputs(
+    $gameTitle,
+    $consoleName,
+);
+
+RenderHtmlStart();
+if ($type == 0) {
```

Comment:
Initially because the general format of the pages were so similar. Doesn't really have any benefit of maintaining two pages in a single file, I've separated them.